domain/services/testing/service.go

```diff
@@ -41,220 +41,198 @@ import (
 	"github.com/juju/juju/internal/services"
 )
 
-// TestingDomainServices provides access to the services required by the apiserver.
-type TestingDomainServices struct {
-	machineServiceGetter     func() *machineservice.WatchableService
-	applicationServiceGetter func() *applicationservice.WatchableService
-}
+type placeholderDomainServices struct{}
 
-// NewTestingDomainServices returns a new registry which uses the provided controllerDB
-// function to obtain a controller database.
-func NewTestingDomainServices() *TestingDomainServices {
-	return &TestingDomainServices{}
+// NewPlaceholderDomainServices returns a new registry which can be used as
+// a placeholder for tests which are required to provide a non-nil registry.
+// Asking the registry for services will return nil values so be careful
+// not to expect the registry to be of any use other than as a placeholder.
+func NewPlaceholderDomainServices() *placeholderDomainServices {
+	return &placeholderDomainServices{}
```

Comment:
I also think this should be deleted as well, wholesale.

Reply:
yeah